src/reporter/index.js

```diff
@@ -172,7 +172,7 @@ export default class Reporter {
             const userAgents = task.browserConnectionGroups.map(group => group[0].userAgent);
             const first      = this.reportQueue[0];
 
-            await this.plugin.reportTaskStart(startTime, userAgents, this.testCount, task.testStructure);
+            await this.plugin.reportTaskStart(startTime, userAgents, this.testCount, task.testStructure, task.opts);
```

Comment:
I think we need to pass an object with the `configuration` field, not a configuration object: ``. It will help us prevent breaking changes if we need to add a new argument in the future.